firefox-ios/Client/Frontend/InternalSchemeHandler/ErrorPageHelper.swift

```diff
@@ -243,10 +243,12 @@ class ErrorPageHelper {
     func loadPage(_ error: NSError, forUrl url: URL, inWebView webView: WKWebView) {
         guard var components = URLComponents(
             string: "\(InternalURL.baseUrl)/\(ErrorPageHandler.path)"
-        ), let webViewUrl = webView.url else { return }
+        ) else { return }
```

Comment:
There's no reason to keep this guard statement across multiple lines, now that it's a single statement.

Reply:
So your suggestion is to remove that guard completely and move the **components** down? (where is used)

Reply:
Sorry, no. Because of the deletion, GH won't let me make a suggestion, but, basically, just a nit. It should read: `guard var components = URLComponents(string: "\(InternalURL.baseUrl)/\(ErrorPageHandler.path)") else { return }`

I really need to get around to writing the custom `guard` rule. lolol

Reply:
Sorry, you mean to make it look like this? :)
<img width="1018" alt="Screenshot 2024-04-04 at 17 21 45" src="https://github.com/mozilla-mobile/firefox-ios/assets/61138287/32ebb587-fa9a-4242-a98e-4069ec44a1da">

Reply:
Did it :)